conans/test/functional/printer_test.py

```diff
@@ -0,0 +1,161 @@
+
+        self.client = TestClient(servers=servers, users={"default": [("lasote", "mypass")]})
+        files = {"conanfile_libA.py": conanfile_libA,
+                 "conanfile_libB.py": conanfile_libB,
+                 "conanfile_libC.py": conanfile_libC,
+                 "txt_file.txt": txt_file,
+                 "test_profile": test_profile
+                }
+        self.client.save(files)
+        self.client.run("export conanfile_libA.py lasote/channel")
+        self.client.run("upload aaa/0.1@lasote/channel -r default")
+        self.client.run("remove -f aaa/0.1@lasote/channel")
+        self.client.run("export conanfile_libB.py lasote/channel")
+        self.client.run("upload bbb/0.1@lasote/channel -r default")
+        self.client.run("remove -f bbb/0.1@lasote/channel")
+        self.client.run("export conanfile_libC.py lasote/channel")
+
+    def print_graph_test(self):
+        self.client.run("install ccc/0.1@lasote/channel",
+                        ignore_error=True) #Ignore ERROR: Missing prebuilt package
+        self.assertIn("aaa/0.1@lasote/channel from 'default'", self.client.out)
+        self.assertIn("bbb/0.1@lasote/channel from 'default'", self.client.out)
+        self.assertIn("ccc/0.1@lasote/channel from local cache", self.client.out)
+
+    def print_info_test(self):
+        self.client.run("info aaa/0.1@lasote/channel")
+
+        aaa_header_output = """aaa/0.1@lasote/channel
+    ID: 5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9
+    BuildID: None
+    Remote: default=http://"""
+        aaa_require_output = """Required by:
+        None"""
+        self.assertIn(aaa_header_output, self.client.out)
+        self.assertIn(aaa_require_output, self.client.out)
+
+        self.client.run("info bbb/0.1@lasote/channel")
+
+        aaa_require_output = """Required by:
+        bbb/0.1@lasote/channel"""
+        bbb_header_output = """bbb/0.1@lasote/channel
+    ID: 05ee4abde05ceb090f7e3a61dd0444da30515355
+    BuildID: None
+    Remote: default=http://"""
+        bbb_require_output = """Required by:
+        None
+    Requires:
+        aaa/0.1@lasote/channel"""
+        self.assertIn(aaa_require_output, self.client.out)
+        self.assertIn(bbb_header_output, self.client.out)
+        self.assertIn(bbb_require_output, self.client.out)
+
+        self.client.run("info ccc/0.1@lasote/channel")
+
+        bbb_require_output = """Required by:
+        ccc/0.1@lasote/channel
+    Requires:
+        aaa/0.1@lasote/channel"""
+        ccc_header_output = """ccc/0.1@lasote/channel
+    ID: 981e05a6766d23b321ba61cd8d473ca507fcfbdc
+    BuildID: None
+    Remote: None"""
+        ccc_require_output = """Required by:
+        None
+    Requires:
+        bbb/0.1@lasote/channel"""
+        self.assertIn(bbb_require_output, self.client.out)
+        self.assertIn(ccc_header_output, self.client.out)
+        self.assertIn(ccc_require_output, self.client.out)
+
+    def print_search_recipes_test(self):
```

Comment:
Too much ``setUp()`` for this simple test. I also think that this functionality should be already in existing tests.

Reply:
These tests only check output messages for printer, some messages are checked in other tests but not all of them. I could reduce the asserts done to keep this smaller